types/auth0/index.d.ts

```diff
@@ -7,8 +7,18 @@
 import * as Promise from 'bluebird';
 
 export interface ManagementClientOptions {
-  token: string;
+  token?: string;
   domain?: string;
```

Comment:
Domain seems to be a mandatory property though, according to the doc you referenced. Can you please remove the question mark on `domain`?